src/Helper/EditorHelper.php

```diff
@@ -46,18 +46,13 @@ public function fromString($string)
             throw new \RuntimeException('No EDITOR environment variable set.');
         }
 
-        if (defined('PHP_WINDOWS_VERSION_MAJOR')) {
```

Comment:
I forgot I removed this as `system($editor.' '.$tmpName.' > `tty`');` is horribly broken 💀